manifests/wego-app/deployment.yaml.tpl

```diff
@@ -13,12 +13,17 @@ spec:
       serviceAccountName: wego-app-service-account
       containers:
         - name: wego-app
-          image: ghcr.io/weaveworks/wego-app:{{.Version}}
-          args: ["ui", "run", "-l"]
+          image: ghcr.io/weaveworks/wego-app:{{.AppVersion}}
+          args: ["ui", "run", "-l", "--helm-repo-namespace", "$(RUNTIME_NAMESPACE)"]
```

Comment:
Should this be a CLI arg if it is already an env variable?

Reply:
I think it makes sense to have it as a CLI flag since we want it to be configurable & have sensible defaults (that someone calling the binary can discover). I'm using the env var as a way of fetching the value. I think its okay? Open to alternatives :smile:

Reply:
An alternative @nikimanoledaki pointed out would be changing:
```
          args: ["ui", "run", "-l", "--helm-repo-namespace", "$(RUNTIME_NAMESPACE)"]
          ports:
            - containerPort: 9001
              protocol: TCP
          imagePullPolicy: IfNotPresent
          env:
            - name: RUNTIME_NAMESPACE
              valueFrom:
                fieldRef:
                  fieldPath: metadata.namespace
```

to:
```
          args: ["ui", "run", "-l", "--helm-repo-namespace", "{{.Namespace}}"]
```

using the templating to insert the namespace instead. I felt env var approach of fetching it from `metadata.namespace` was a tiny bit cleaner, but if the env var adds too much noise to the file then I'm happy to change :smile:

Reply:
Decided to go with the templating. Resolved https://github.com/weaveworks/weave-gitops/pull/1148/commits/149d528bafca07e0581d75f89249f9d52ef7adca